tests/datasource/data_connector/test_configured_asset_azure_data_connector.py

```diff
@@ -0,0 +1,1166 @@
+import os
+
+import pytest
+from azure.storage.blob import BlobServiceClient, ContainerClient
+
+from great_expectations.datasource.data_connector import (
+    ConfiguredAssetAzureDataConnector,
+)
+from great_expectations.datasource.data_connector.util import list_azure_keys
+
+try:
+    ACCOUNT_URL = os.environ["ACCOUNT_URL"]
+    CONTAINER_NAME = os.environ["CONTAINER_NAME"]
+except:
+    print("Please set environment variables before running tests")
```

Comment:
This should all come from the data doc link I've shared on Confluence. @petermoyer asked for us to draft a spec or general design doc to improve knowledge sharing so that would be a good place to update as we progress with this.

Reply:
Note that env vars should come from `great_expectations/uncommitted/config_variables.yml`